src/hyperlight_host/src/sandbox/outb.rs

```diff
@@ -117,7 +99,7 @@ fn handle_outb_impl(
     mem_mgr: &mut MemMgrWrapper<HostSharedMemory>,
     host_funcs: Arc<Mutex<HostFuncsWrapper>>,
     port: u16,
-    byte: u64,
+    data: Vec<u8>,
```

Comment:
I'm not sure I understand why this changed. Do I understand correctly that this vec will only ever have length 1?

Reply:
It can be up to 4 bytes now

Reply:
I changed the outb instruction to now send 4 bytes instead of 1. This is useful for less exits w/ `debug_print` and to also be able to send more information w/ exceptions.